src/bokeh/models/widgets/tables.py

```diff
@@ -886,6 +886,10 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
     The height of each row in pixels.
     """)
 
+    multi_selectable = Bool(True, help="""
+    Whether multiple rows can be selected.
+    """)
```

Comment:
I'm not sure about this naming convention. I would probably have called this `multi_selection = Bool(...)`, though we also have plenty of `*able` properties. I will defer this to other to decide. Otherwise this PR looks good, though I wouldn't hurt to add an integration test.

Reply:
I don't have any strong opinions here. @tcmetzger ?